tests/std/tests/P0896R4_ranges_alg_sort/test.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <array>
+#include <cassert>
+#include <concepts>
+#include <ranges>
+#include <utility>
+
+#include <range_algorithm_support.hpp>
+
+using namespace std;
+using P = pair<int, int>;
+
+// Validate dangling story
+STATIC_ASSERT(same_as<decltype(ranges::sort(borrowed<false>{})), ranges::dangling>);
+STATIC_ASSERT(same_as<decltype(ranges::sort(borrowed<true>{})), int*>);
+
+struct instantiator {
+    static constexpr array input = {P{-1200257975, 0}, P{-1260655766, 1}, P{-1298559576, 2}, P{-1459960308, 3},

```

Comment:
Do we need array here or can we increase throughput by using a plain C-style array

Reply:
Replacing this with a C array - and replacing the two copy constructions in the body of the test with calls to `ranges::copy` - is a wash on my machine. Fastest of ten runs of all 17 configurations is 5.52s (`std::array`) vs. 5.55s (C array).

The `array` initialization is uglier - the `P`s are necessary for CTAD to work - but the copy construction syntax at the use sites is superior. so readability is a bit of a wash as well. I see no reason to prefer either approach so I'll stick with the status quo.